setup.sh

```diff
@@ -0,0 +1,382 @@
+        exit 0
+        ;;
+      *)
+        echo
+        writeBold "$RED" "[✖] I did not understand your answer."
+        ;;
+    esac
+  done
+}
+
+installDockerCompose() {
+  echo
+  writeBold "[ℹ] Docker Compose must be installed to run Kuzzle."
+  writeBold "    This script can install Docker Compose for you, otherwise you can do it manually."
+  write     "    More information at https://docs.docker.com/compose/install/"
+  while [[ "$installDockerCompose" != [yYnN] ]]
+  do
+    promptBold "[❓] Do you want to install Docker Compose now? (y/N)"
+    read installDockerCompose trash
+    case "$installDockerCompose" in
+      [yY])
+        if commandExists curl; then
+          echo
+          writeBold "Installing Docker Compose..."
+          curl -L "https://github.com/docker/compose/releases/download/1.10.0/docker-compose-$(uname -s)-$(uname -m)" -o $DOCKER_COMPOSE_BIN
+          chmod +x $DOCKER_COMPOSE_BIN
+          echo
+          writeBold "$GREEN" "[✔] Docker Compose successfully installed."
+        elif commandExists wget; then
+          echo
+          writeBold "Installing Docker Compose..."
+          wget -O $DOCKER_COMPOSE_BIN "https://github.com/docker/compose/releases/download/1.10.0/docker-compose-$(uname -s)-$(uname -m)"
+          chmod +x $DOCKER_COMPOSE_BIN
+          echo
+          writeBold "$GREEN" "[✔] Docker Compose successfully installed."
+        else
+          writeBold "$RED" "[✖] curl or wget need to be installed to launch the Docker Compose installation script,"
+          writeBold "$RED" "    but none seems to be installed on your system."
+          echo
+          writeBold "$BLUE" "Please install curl or wget and re-run this script."
+          echo
+          exit 3
+        fi
+        ;;
+      [nN] | '')
+        echo
+        writeBold "$BLUE" "Ok. Please install Docker Compose and re-run this script."
+        echo
+        exit 0
+        ;;
+      *)
+        echo
+        writeBold "$RED" "[✖] I did not understand your answer."
+        ;;
+    esac
+  done
+}
+
+setupMapCount() {
+  SYSCTL_CONF_FILE=/etc/sysctl.conf
+  echo
+  writeBold "[ℹ] The kernel configuration variable vm.max_map_count must be set to at least $REQUIRED_MAP_COUNT"
+  writeBold "    for Kuzzle to work properly, but it seems to be set to $MAP_COUNT on your system."
+  writeBold "    This script can set it automatically or you can do it manually."
+  write     "    More information at https://www.elastic.co/guide/en/elasticsearch/reference/5.x/vm-max-map-count.html"
+  while [[ "$setVmParam" != [yYnN] ]]
+  do
+    promptBold "[❓] Do you want to set the vm.max_map_count now? (y/N) "
+    read setVmParam trash
+    case "$setVmParam" in
+      [yY])
+        echo
+        writeBold "Setting kernel variable vm.max_map_count to $REQUIRED_MAP_COUNT..."
+        sysctl -w vm.max_map_count=$REQUIRED_MAP_COUNT
+        if [ -z "$MAP_COUNT" ]; then
+          echo "vm.max_map_count=$REQUIRED_MAP_COUNT" > $SYSCTL_CONF_FILE
```

Comment:
I believe you want to **add** the configuration in the sysctl configuration file, not overwrite it with this config only:

`echo "vm.max_map_count=$REQUIRED_MAP_COUNT" >> $SYSCTL_CONF_FILE`